climpred/comparisons.py

```diff
@@ -204,6 +204,8 @@ def _m2c(ds, control_member=None, metric=None):
     forecast = _drop_members(ds, removed_member=ds.member.values[control_member])
     if not metric.probabilistic:
         forecast, reference = xr.broadcast(forecast, reference)
+    elif 'member' in reference.coords:
```

Comment:
I think this is just supposed to be `if` and not `elif`? I don't think it's related to the probabilistic condition.

Reply:
the problem occured only for probabilistic. and other tests fail for `if`